plugins/source/aws/resources/services/inspector2/coverages.go

```diff
@@ -0,0 +1,54 @@
+package inspector2
+
+import (
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
```

Comment:
If you apply helper columns from `client` package & the `MaxResults` fix, the imports will need to be changed:
```suggestion
	"context"

	"github.com/aws/aws-sdk-go-v2/aws"
```